libnetdata/socket/security.c

```diff
@@ -30,13 +30,13 @@ void security_openssl_library()
 
```

Comment:
Also remove the first and last space inside the `()` : e.g. `if ( !ctx ) -> if (!ctx)`

Reply:
The last pull fixed  the spaces as you requested!